src/rgw/rgw_common.cc

```diff
@@ -2508,12 +2552,14 @@ void RGWBucketInfo::dump(Formatter *f) const
   encode_json("creation_time", ut, f);
   encode_json("owner", owner.to_str(), f);
   encode_json("flags", flags, f);
+  encode_json("flags_interpreted", dump_flags(), f);
   encode_json("zonegroup", zonegroup, f);
   encode_json("placement_rule", placement_rule, f);
   encode_json("has_instance_obj", has_instance_obj, f);
   encode_json("quota", quota, f);
   encode_json("num_shards", layout.current_index.layout.normal.num_shards, f);
   encode_json("bi_shard_hash_type", (uint32_t)layout.current_index.layout.normal.hash_type, f);
+  encode_json("current_index_layout_generation", get_current_index().gen, f);
```

Comment:
there's a `radosgw-admin bucket layout` command that dumps the local layout information in detail

because this RGWBucketInfo json format is used by multisite metadata sync, we intentionally omitted the `layout` information that can differ between zones